grafeas/objects/vulnerability.go

```diff
@@ -45,29 +45,26 @@ type VulnerabilityDetails struct {
 func (vd *VulnerabilityDetails) AsVoucherVulnerability(noteName, vulProject string) (vul voucher.Vulnerability) {
 	vul.Name = strings.Replace(noteName, vulProject, "", 1)
 
-	vul.Severity = getSeverity(vd.Severity)
-	packageIssues := vd.PackageIssue
-
-	if vul.Severity == voucher.UnknownSeverity && len(packageIssues) > 0 {
-		vul.Severity = getSeverity(vd.EffectiveSeverity)
-	}
+	vul.Severity = getSeverity(vd.EffectiveSeverity)
 
 	return
 }
 
 // getSeverity translates the client-fo grafeas Severity to a Voucher Severity.
 func getSeverity(severity *VulnerabilitySeverity) voucher.Severity {
-	switch *severity {
-	case SeverityMinimal:
-		return voucher.NegligibleSeverity
-	case SeverityLow:
-		return voucher.LowSeverity
-	case SeverityMedium:
-		return voucher.MediumSeverity
-	case SeverityHigh:
-		return voucher.HighSeverity
-	case SeverityCritical:
-		return voucher.CriticalSeverity
+	if severity != nil { 
+		switch *severity {
+		case SeverityMinimal:
+			return voucher.NegligibleSeverity
+		case SeverityLow:
+			return voucher.LowSeverity
+		case SeverityMedium:
+			return voucher.MediumSeverity
+		case SeverityHigh:
+			return voucher.HighSeverity
+		case SeverityCritical:
+			return voucher.CriticalSeverity
+		}
 	}
 	return voucher.UnknownSeverity
```

Comment:
instead of nesting this all under 1 `if` can we instead have a quicker return above?

```suggestion
	if severity == nil { 
	    	return voucher.UnknownSeverity
	}
	switch *severity {
	case SeverityMinimal:
		return voucher.NegligibleSeverity
	case SeverityLow:
		return voucher.LowSeverity
	case SeverityMedium:
		return voucher.MediumSeverity
	case SeverityHigh:
		return voucher.HighSeverity
	case SeverityCritical:
		return voucher.CriticalSeverity
	}
```